open_gpt/models/modeling.py

```diff
@@ -57,9 +62,23 @@ def load_model_and_transforms(
             device_map=self._device_map,
         )
 
+        if adapter_name_or_path is not None:
+            self.load_adapter(adapter_name_or_path)
+
         # turn the eval mode off `eval_mode=False` in training
         if self._eval_mode:
             self.model.eval()
 
     def post_init(self, **kwargs):
         pass
+
+    def load_adapter(self, adapter_name_or_path: str):
+        from peft import PeftModel
+
+        self.model = PeftModel.from_pretrained(
+            self.model,
+            adapter_name_or_path,
+            device_map={'': self._device or 0}
```

Comment:
`device_map` is also supported in peft?

Reply:
yes, but it's in `kwargs`, will be used in `load_adapter`: 
<img width="903" alt="image" src="https://github.com/jina-ai/opengpt/assets/16580382/f5cc908a-4ecc-4959-a3bc-7ab5fd37dde9">